wordpress.org/public_html/wp-content/themes/pub/wporg-developer/inc/user-content.php

```diff
@@ -466,6 +468,8 @@ public static function get_editor_rules( $context = '', $note_link = '' ) {
 		return '<ul><li>'
 			. __( 'This form is not for support requests, discussions, spam, bug reports, complaints, or self-promotion. Entries of this nature will be deleted.', 'wporg' )
 			. '</li><li>'
+			. __( 'Notes and feedback must be written in English. Entries not written in English will be deleted.', 'wporg' )
```

Comment:
Same as above.

Take note that I'm aware the line that discourages spam / support requests says delete. I think those cases warrant a stronger line.